test/system/test_correct_genotypes_by_imputation.py

```diff
@@ -0,0 +1,87 @@
+import json
+import os
+import subprocess
+from test import get_resource_dir, test_dir
+
+import pandas as pd
+import numpy as np
+
+from ugvc.pipelines import correct_genotypes_by_imputation as corgen
+from ugvc.vcfbed.vcftools import genotype_ordering
+
+class TestCorrectGenotypesByImputation:
+    inputs_dir = get_resource_dir(__file__)
+    resources_dir = get_resource_dir(__file__)
+
+    def test_convert_ds_to_genotype_imputation_priors(self):
+        # Homozygous biallelic variant
+        ds_ar = np.array([2])
+        gt_ar = genotype_ordering(1)
+        num_alt = 1
+        epsilon = 0.01
+        assert np.all(corgen._convert_ds_to_genotype_imputation_priors(ds_ar, gt_ar, num_alt, epsilon) == np.array([1,0.01,0.99]))
+        # Heterozygous biallelic variant
+        ds_ar = np.array([1])
+        gt_ar = genotype_ordering(1)
```

Comment:
this 1 is num_alt defined below.
It would be good to use the variable instead of a constant